src/fu-security-attr.c

```diff
@@ -207,6 +207,314 @@ fu_security_attr_get_name(FwupdSecurityAttr *attr)
+	}
+	if (g_strcmp0(appstream_id, FWUPD_SECURITY_ATTR_ID_INTEL_SMAP) == 0) {
+		/* TRANSLATORS: Title: SMAP = Supervisor Mode Access Prevention */
+		return _("Intel SMAP");
+	}
+	if (g_strcmp0(appstream_id, FWUPD_SECURITY_ATTR_ID_ENCRYPTED_RAM) == 0) {
+		/* TRANSLATORS: Title: Memory contents are encrypted, e.g. Intel TME */
+		return _("Encrypted RAM");
+	}
+	if (g_strcmp0(appstream_id, FWUPD_SECURITY_ATTR_ID_IOMMU) == 0) {
+		/* TRANSLATORS: Title:
+		 * https://en.wikipedia.org/wiki/Input%E2%80%93output_memory_management_unit */
+		return _("IOMMU");
+	}
+	if (g_strcmp0(appstream_id, FWUPD_SECURITY_ATTR_ID_KERNEL_LOCKDOWN) == 0) {
+		/* TRANSLATORS: Title: lockdown is a security mode of the kernel */
+		return _("Linux Kernel Lockdown");
+	}
+	if (g_strcmp0(appstream_id, FWUPD_SECURITY_ATTR_ID_KERNEL_TAINTED) == 0) {
+		/* TRANSLATORS: Title: if it's tainted or not */
+		return _("Linux Kernel Verification");
+	}
+	if (g_strcmp0(appstream_id, FWUPD_SECURITY_ATTR_ID_KERNEL_SWAP) == 0) {
+		/* TRANSLATORS: Title: swap space or swap partition */
+		return _("Linux Swap");
+	}
+	if (g_strcmp0(appstream_id, FWUPD_SECURITY_ATTR_ID_SUSPEND_TO_RAM) == 0) {
+		/* TRANSLATORS: Title: sleep state */
+		return _("Suspend To RAM");
+	}
+	if (g_strcmp0(appstream_id, FWUPD_SECURITY_ATTR_ID_SUSPEND_TO_IDLE) == 0) {
+		/* TRANSLATORS: Title: a better sleep state */
+		return _("Suspend To Idle");
+	}
+	if (g_strcmp0(appstream_id, FWUPD_SECURITY_ATTR_ID_UEFI_PK) == 0) {
+		/* TRANSLATORS: Title: PK is the 'platform key' for the machine */
+		return _("UEFI Platform Key");
+	}
+	if (g_strcmp0(appstream_id, FWUPD_SECURITY_ATTR_ID_UEFI_SECUREBOOT) == 0) {
+		/* TRANSLATORS: Title: SB is a way of locking down UEFI */
+		return _("UEFI Secure Boot");
+	}
+	if (g_strcmp0(appstream_id, FWUPD_SECURITY_ATTR_ID_TPM_EMPTY_PCR) == 0) {
+		/* TRANSLATORS: Title: PCRs (Platform Configuration Registers) shouldn't be empty */
+		return _("TPM Empty PCRs");
+	}
+	if (g_strcmp0(appstream_id, FWUPD_SECURITY_ATTR_ID_TPM_RECONSTRUCTION_PCR0) == 0) {
+		/* TRANSLATORS: Title: the PCR is rebuilt from the TPM event log */
+		return _("TPM PCR0 Reconstruction");
+	}
+	if (g_strcmp0(appstream_id, FWUPD_SECURITY_ATTR_ID_TPM_VERSION_20) == 0) {
+		/* TRANSLATORS: Title: TPM = Trusted Platform Module */
+		return _("TPM v2.0");
+	}
+	if (g_strcmp0(appstream_id, FWUPD_SECURITY_ATTR_ID_MEI_MANUFACTURING_MODE) == 0) {
+		/* TRANSLATORS: Title: MEI = Intel Management Engine */
+		return _("MEI Manufacturing Mode");
+	}
+	if (g_strcmp0(appstream_id, FWUPD_SECURITY_ATTR_ID_MEI_OVERRIDE_STRAP) == 0) {
+		/* TRANSLATORS: Title: MEI = Intel Management Engine, and the "override" is enabled
+		 * with a jumper -- luckily it is probably not accessible to end users on consumer
+		 * boards */
+		return _("MEI Override");
+	}
+	if (g_strcmp0(appstream_id, FWUPD_SECURITY_ATTR_ID_MEI_VERSION) == 0) {
+		/* TRANSLATORS: Title: MEI = Intel Management Engine */
+		return _("MEI Version");
+	}
```

Comment:
Similar to my SPI comment, I think you should expand this to "Intel Management Engine Interface ..."

Reply:
Added to the google doc

Reply:
I think the google doc is no longer maintained right?   Shouldn't it go into the markdown document now?

Reply:
I've sent you a link to the gdoc.

Reply:
Wouldn't it have been easier to just review it all here in one place? :)

Reply:
Ya, I'm tempted to push what we have and then iterate on specific strings when we can see then in an actual UI.